contributing/index.md

```diff
@@ -55,6 +55,26 @@ Please help us:
     (No new features just yet!)
 
 
+## Writing or Modifying GN Targets
+
+We provide the following [GN Templates](https://chromium.googlesource.com/chromium/src/+/master/tools/gn/docs/language.md#Templates)
+to ensure that all our [targets](https://chromium.googlesource.com/chromium/src/+/master/tools/gn/docs/language.md#Targets)
+are built with the same configuration:
+
+  * `rtc_test` which replaces `test`
+  * `rtc_source_set` which replaces `source_set`
+  * `rtc_executable` which replaces  executable 
+  * `rtc_static_library` which replaces `static_library`
+
+All templates include both [`common_config`](https://cs.chromium.org/chromium/src/third_party/webrtc/BUILD.gn?rcl=0&l=105)
```

Comment:
Please remove the line-number part of the links, since they'll quickly become outdated. 
I think it's still nice to link to the BUILD.gn file (then the reader has to find the target himself).